lib/types/types.ts

```diff
@@ -22,9 +22,9 @@ export type PrimitiveType =
 export type OriginalType =
 // Converted Types
     | 'UTF8' // 0
-    // | 'MAP' // 1
+    | 'MAP' // 1
     // | 'MAP_KEY_VALUE' // 2
-    // | 'LIST' // 3
+    | 'LIST' // 3
```

Comment:
I had to uncomment these two type values in order for a line in schema to remain valid. I'm not sure if this is the right choice because it could be commented for a reason.